vite.config.ts

```diff
@@ -5,6 +5,7 @@ import dts from 'vite-plugin-dts';
 // https://vitejs.dev/guide/build.html#library-mode
 export default defineConfig({
   build: {
+    ssr: true,
```

Comment:
otherwise vite builds for browser compatibility (in general even?) and fails the build with
```
"createHmac" is not exported by "__vite-browser-external"
```

As far as I understand this https://github.com/vitejs/vite/issues/13926#issuecomment-1686013974, vite is not even supposed to work for node libraries?